core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -19,48 +19,55 @@
 import static java.util.Objects.requireNonNull;
 
 import java.util.concurrent.ThreadFactory;
+import java.util.function.Function;
 
 import javax.annotation.Nullable;
 
-import io.netty.util.concurrent.DefaultThreadFactory;
-
 /**
  * {@link ThreadFactory} that creates event loop threads.
  *
  * @see EventLoopGroups
+ *
+ * @deprecated Use {@link ThreadFactories#newEventLoopThreadFactory(String, boolean)} or
+ * {@link ThreadFactories#builder(String)}. Note that setting
+ * {@link ThreadFactoryBuilder#eventLoop(boolean)} true is required to create event loop thread. e.g.
+ * {@code ThreadFactories.builder("treadPrefix").eventLoop(true).build()}.
```

Comment:
`treadPrefix` -> `myThreadPrefix` ? (missing `h`)

Reply:
I fixed it `myThreadNamePrefix`. Because it looks more consistent.

Over the project, the name `threadNamePrefix` is used except one case.
I also renamed the case `threadPrefix` to `threadNamePrefix`.
Is it okay?

Reply:
Yes. Thanks!